pymatgen/entries/compatibility.py

```diff
@@ -664,6 +641,13 @@ def __init__(self, corrections: Sequence[Correction]):
     def get_adjustments(self, entry: AnyComputedEntry) -> list[EnergyAdjustment]:
         """Get the list of energy adjustments to be applied to an entry."""
         adjustment_list = []
+        # only correct GGA or GGA+U entries
+        if entry.parameters.get("run_type") not in ("GGA", "GGA+U"):
+            raise CompatibilityError(
+                f"Entry {entry.entry_id} has invalid run type {entry.parameters.get('run_type')}. "
+                f"Must be GGA or GGA+U. Discarding."
+            )
+
```

Comment:
Actually, since this method is in the base `CorrectionsList` class and not specifically in `MaterialsProjectCompatibility`, I need to add a kwarg to specify what the valid run types are in case there are other compatibility classes derived from this one. Will do that next week.